doc/source/tutorials/basic_usage.rst

```diff
@@ -113,7 +120,7 @@ are described in both the
 should be an essential route in to conducting simulations with the martini force field.
 
 We cover the documentation of these features in greater detail in the pages about
-`Elastic Networks <elastic_networks.rst>`_ and `Gō models <go_models.rst>`_.
+`Elastic Networks </tutorials/elastic_networks.html>`_ and `Gō models </tutorials/go_models.html>`_.
```

Comment:
This feels off, because these rst files could also be rendered to pdf. See https://github.com/marrink-lab/vermouth-martinize/blob/master/doc/source/martinize2_workflow.rst?plain=1#L150 for an example